### PR TITLE
Create spelling-grammar.rst

### PR DESCRIPTION
**Pull request by mechanomi**
this PR creates a new spelling and grammar style guide based on work done internally by marketing

feedback and criticism is very welcome

### CODE REVIEW

File: style/spelling-grammar.rst [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+
+  - Exceptions to the rule:
+  
+    - Write "ad hoc" instead of "ad-hoc". (Citation: `StackExchange <https://english.stackexchange.com/a/157332>`_.)
+
+- Drop the hypen from compound words when it is common practice to do so.
+
+  - Examples:
+
+    - Write "database". Avoid "data-base" and "data base".
```
Comment (infoverload) [inline style/spelling-grammar.rst:45]: Haha I've never seen "data-base" or "data base" before but I guess it can happen !

**Review by infoverload — APPROVED** [review]
looks ok

**Comment by amotl** [comment]
Thanks for this!

**Comment by matthijskrul** [comment]
Good work!

**Comment by carlotasoto** [comment]
Thank you @mechanomi !

**Pull request merged**